src/styles/colors.js

```diff
@@ -18,4 +18,5 @@ export default {
     redDisabled: '#fea29a',
     yellow: '#fed607',
     transparent: 'transparent',
+    translucentWhite: 'rgba(255,255,255, 0.96)',
```

Comment:
Rather than needing to add a new color value here, can we do something similar to what we've done for the modal background? Basically we just use the standard white color, but then add opacity as a separate style?

Reply:
Actually just realized I contradicted this review comment in a later review too. Sorry about the confusion